bitnami/grafana/values-production.yaml

```diff
@@ -277,3 +277,38 @@ affinity: {}
 ## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
 ##
 podAnnotations: {}
+
+## Prometheus metrics
+##
+metrics:
+  enabled: false
+
+  ## Prometheus Operator ServiceMonitor configuration
+  ##
+  service:
+    annotations:
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "{{ .Values.service.port }}"
+      prometheus.io/path: "/metrics"
```

Comment:
Where are you using these `metrics.service.annotations`? It seems there's no metrics service using them

Reply:
fixed!